publicroomsapi/directory/directory.go

```diff
@@ -54,11 +57,50 @@ func GetVisibility(
 }
 
 // SetVisibility implements PUT /directory/list/room/{roomID}
-// TODO: Check if user has the power level to edit the room visibility
```

Comment:
Perhaps can be replaced with a new TODO that talks about the to-be-added admin privileges :)

Reply:
done